projects/gloo/cli/pkg/cmd/check/gloo_stats.go

```diff
@@ -51,6 +51,12 @@ func ResourcesSyncedOverXds(stats, deploymentName string) bool {
 		fmt.Println(resourcesOutOfSyncMessage(outOfSyncResources))
 		return false
 	}
+
+	if len(metrics) == 0 {
+		fmt.Println("No xds metrics to check")
+	} else {
+		fmt.Println("OK")
+	}
```

Comment:
This will break glooctl output types right? I think we do that in the out of sync but this will make it always escape the output formats

Reply:
Good call. How about now?